From 14c29c8bba69da9467c8facbc5b2137b3fe1bead Mon Sep 17 00:00:00 2001
From: Vishal Kumar Pal <quic_vispal@quicinc.com>
Date: Fri, 27 Jun 2025 11:19:19 +0530
Subject: [PATCH 1/2] PENDING: arm64: dts: qcom: Add EEPROM support for
 IQ-9075-evk

Add dts change to support EEPROM on IQ-9075-evk.

Change-Id: I020078aa0011c0f8922d71047b81fe2d6fe4803a
Signed-off-by: Vishal Kumar Pal <quic_vispal@quicinc.com>
Upstream-Status: Pending

---
 arch/arm64/boot/dts/qcom/qcs9075-iq-9075-evk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/qcs9075-iq-9075-evk.dts b/arch/arm64/boot/dts/qcom/qcs9075-iq-9075-evk.dts
index 763208149fe1..9e80d1c0fb94 100644
--- a/arch/arm64/boot/dts/qcom/qcs9075-iq-9075-evk.dts
+++ b/arch/arm64/boot/dts/qcom/qcs9075-iq-9075-evk.dts
@@ -334,6 +334,7 @@ &ethernet0 {
 	snps,mtl-rx-config = <&mtl_rx_setup>;
 	snps,mtl-tx-config = <&mtl_tx_setup>;
 	snps,ps-speed = <1000>;
+	eeprom-for-mac = <&eeprom0>;
 
 	status = "okay";
 
-- 
2.25.1

